applications/blockchainia.md

```diff
@@ -0,0 +1,139 @@
+
+These repos include our R & D work, although we have not begun development on our first iteration of the MVP. We will be starting two projects from scratch, the Blockchainia blockchain, and the DeathToll vide game, for Milestone 1. These repos will include all necessary documentation to run and test our functionality.
+
+- https://github.com/eca20/BlockchainiaJRPG
+- https://github.com/eca20/SubstratePlayGround
+- https://github.com/eca20/Blockchainia-Pitch
+
+
+ Development Roadmap :nut_and_bolt:
+
+ ### Overview
+
+ - **Total Estimated Duration:** Continuous
+ - **Full-Time Equivalent (FTE):**  2
+ - **Total Costs:** $25,000
+
+ ### Milestone 1  — Basic MVP
+
+ - **Estimated duration:** 13 weeks development 
+ - **FTE:**  .97 FTE
+ - **Costs:** $25,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 / GPLv3 / MIT / Unlicense |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. Unit tests will be written for relevent game server functionality. Unit tests will be written for the DeathToll pallet. In each guide (in each project's Github repo), we will describe how to run these tests. Integration testing will be enabled through game functionality (i.e, Given an enemy is eliminated on server, the proper state functions are executed on chain, and the leaderboard updates through the in-game chain browser) |
+| **0d.** | Docker | We will provide Dockerfiles that can be used to run and test all the functionality delivered with this milestone. This will include a deployable "Full Node" that encases both a game server and substrate node. These will interact with our game client and substrate parachain respectively. A successful MVP will allow a game client to register events on the game server (Server-Authoritative model), which is then written to the parachain ledger. Upon completion of the initial game "level", an "accolade" NFT will be deposited via ink! smart contract to the wallet that completed the required in-game tasks. |
+| 1. | Social Media | We will deploy a social media marketing campaign upon the successful development and testing of an MVP. This will include a social media campaign, building a discord community of evangelists, and marketing materials that describe our project to potential stakeholders and various customers of our ecosystem. The first milestone will involve hitting 1000 followers, which will enable us to involve our community from Milestone 2 forward. 
```

Comment:
This is also [not covered](https://w3f.github.io/Grants-Program/docs/faq#what-activitiespositions-do-you-fund) by the grants program. Please remove.